eZ/Publish/Core/Persistence/Cache/ContentHandler.php

```diff
@@ -21,10 +21,54 @@
 /**
  * @see \eZ\Publish\SPI\Persistence\Content\Handler
  */
-class ContentHandler extends AbstractHandler implements ContentHandlerInterface
+class ContentHandler extends AbstractInMemoryPersistenceHandler implements ContentHandlerInterface
 {
     const ALL_TRANSLATIONS_KEY = '0';
 
+    /** @var callable */
+    private $getContentInfoTags;
+
+    /** @var callable */
+    private $getContentInfoKeys;
+
+    /** @var callable */
```

Comment:
Please use multi-line doc for members, for consistency's sake. We'll drop them completely once we switch to PHP 7.4 in eZ Platform 4.0